.gitignore

```diff
@@ -131,3 +131,6 @@ sponge-theme/
 
 # Source code created by pip
 src/
+
+# compiled sponge-javadoc extension
+etc/sponge-javadoc/sponge-javadoc.pyc
```

Comment:
Hmm, seems its covered above by `*.py[cod]` anyway. Guess I'll just remove this one then.